File: app/metier/send_mail.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart
from app.metier.variables import Variables


def send_message(to, subject, info):
    try:
        msg = MIMEMultipart()
        msg['From'] = "no-reply@cirt.cm"
        msg['To'] = to
        msg['Bcc'] = Variables.WANKI_EMAIL
        msg['Subject'] = subject
        msg.attach(MIMEText(info, 'plain'))

        server = smtplib.SMTP("mail.cirt.cm", 25)
        server.login("h.wanki@cirt.cm", "Alu1234!@#$1990")
        server.sendmail("no-reply@cirt.cm", [to, Variables.WANKI_EMAIL], msg.as_string())
        server.quit()
        print("Done")
    except Exception as e:
        raise RuntimeError(str(e))

File: src/main/java/cm/cirt/calldatarecordmanagement/metier/SendMail.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package cm.cirt.calldatarecordmanagement.metier;

import java.util.Properties;
import javax.mail.Message;
import javax.mail.MessagingException;
import javax.mail.PasswordAuthentication;
import javax.mail.Session;
import javax.mail.Transport;
import javax.mail.internet.InternetAddress;
import javax.mail.internet.MimeMessage;

/**
 *
 * @author Wanki
 */
public class SendMail {

    public static void sendMessage(String to, String Subject, String info) {
        Properties props = new Properties();
        props.put("mail.smtp.host", "mail.cirt.cm");
        props.put("mail.smtp.socketFactory.port", "25");
        props.put("mail.smtp.socketFactory.class",
                "javax.net.ssl.SSLSocketFactory");
        props.put("mail.smtp.auth", "true");
        props.put("mail.smtp.port", "25");

        Session session = Session.getDefaultInstance(props,
                new javax.mail.Authenticator() {
            @Override
            protected PasswordAuthentication getPasswordAuthentication() {
                return new PasswordAuthentication("h.wanki@cirt.cm", "Alu1234!@#$1990");
            }
        });

        try {

            Message message = new MimeMessage(session);
            message.setFrom(new InternetAddress("no-reply@cirt.cm"));
            message.setRecipients(Message.RecipientType.TO,
                    InternetAddress.parse(to));
            message.setRecipients(Message.RecipientType.BCC,
                    InternetAddress.parse(Variables111.WANKI_EMAIL));
            message.setSubject(Subject);
            message.setText(info);

            Transport.send(message);

            System.out.println("Done");

        } catch (MessagingException e) {
            throw new RuntimeException(e);
        }
    }
    
    public static void main(String[] args) {
        SendMail.sendMessage("h.wanki@cirt.cm", "Just a test", "Hello \n Just an email test.");
    }
}
